packages/modules/src/BaseValidationModule.ts

```diff
@@ -28,7 +27,7 @@ export abstract class BaseValidationModule implements IValidationModule {
   // Review naming convention for getter
   abstract getSigner(): Promise<Signer>
 
-  abstract signUserOp(userOperation: UserOperation): Promise<string>
+  abstract signUserOp(userOperation: UserOperation, sessionSigner?: Signer): Promise<string>
```

Comment:
can we just name this a signer in abstract class. makes more sense

Reply:
or you can just omit it. if in child class name to be kept as sessionSigner only.
cc @tomarsachin2271